### PR TITLE
ComboBox: Scroll selected option to top

### PR DESCRIPTION
**Pull request by kellygorr**
#### Pull request checklist

 [0] Addresses an existing issue: Fixes #0000
 [X] Include a change request file using `$ npm run change`

#### Description of changes
Added new prop.  When options in ComboBox are scrollable, scroll selected option to top of callout (unless it has reached the end of the scrollbar).

Before

After

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.Props.ts [hunk 1]
```diff
@@ -88,6 +88,13 @@ export interface IComboBoxProps extends ISelectableDroppableTextProps<IComboBox>
    * in case an option does not come with user-defined custom styles
    */
   comboBoxOptionStyles?: Partial<IComboBoxOptionStyles>;
+
+  /**
+   * When options are scrollable the selected option is positioned at the top of the callout when it is opened (unless it has reached the end of the scrollbar).
+   *
```
Comment (micahgodbolt) [inline packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.Props.ts]: please add a @default false here

**Comment by micahgodbolt** [comment]
@kellygorr out of curiosity, do you have any idea why your PRs so often have these big long lists of master merges?

image

**Comment by kellygorr** [comment]
@micahgodbolt - I don't know why (I wish it didn't).  When I create a branch it brings over all of the commits that have ever been done on the master.  I use SourceTree, so maybe there is something I have checked when I create a branch.

**Comment by micahgodbolt** [comment]
@kellygorr hmmm, yeah, maybe a source tree thing. You might try doing `git checkout -b new-branch` from master sometime and see if you have different results

**Pull request merged**